bin/strata-test-cli/src/cmd/create_deposit_tx.rs

```diff
@@ -0,0 +1,58 @@
+use argh::FromArgs;
+use strata_cli_common::errors::{DisplayableError, DisplayedError};
+
+use crate::bridge::dt;
+
+/// Arguments for creating a deposit transaction.
+///
+/// Creates a deposit transaction from a Deposit Request Transaction (DRT) using operator keys.
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "create-deposit-tx")]
+pub struct CreateDepositTxArgs {
+    #[argh(option)]
+    /// raw DRT transaction in hex format
+    pub drt_tx: String,
+
+    #[argh(option)]
+    /// operator private keys in JSON array format (each key is 78 bytes hex)
```

Comment:
Can you give some examples in the docstring.
This is ambiguous.
Is it

```console
--operator-keys='["foo", "bar"]'
```

or

```console
--operator-keys='{"operator_keys": ["foo", "bar"]}'
```

see my point?